target/linux/lantiq/falcon/config-5.4

```diff
@@ -7,3 +7,4 @@ CONFIG_PINCTRL_FALCON=y
 CONFIG_SOC_FALCON=y
 # CONFIG_SOC_XWAY is not set
 CONFIG_SPI_FALCON=y
+CONFIG_SWCONFIG=y
```

Comment:
You can remove the CONFIG_SWCONFIG from Falcon, the chip does not have a switch.